src/SFA.DAS.AssessorService.Data/CertificateRepository.cs

```diff
@@ -300,40 +300,129 @@ public async Task UpdateCertificatesReadyToPrintInBatch(Guid[] certificateIds, i
+                .Skip((pageIndex - 1) * pageSize)
+                .Take(pageSize).ToListAsync();
+
+            var count = await certificatesQuery.Select(x => x.Certificate.Id).CountAsync();
 
             return new PaginatedList<Certificate>(certificates, count, pageIndex, pageSize);
         }
 
+        private IQueryable<SortableCertificateInfo> GetSortedCertificateInfo(bool sortDescending, GetCertificateHistoryRequest.SortColumns sort, IQueryable<SortableCertificateInfo> certificatesQuery)
+        {
+            switch (sort)
+            {
+                case GetCertificateHistoryRequest.SortColumns.Apprentice:
+
+                    if (sortDescending)
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderByDescending(x => x.CertificateData.FullName);
+                    }
+                    else
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderBy(x => x.CertificateData.FullName);
+                    }
+
+                    break;
+                case GetCertificateHistoryRequest.SortColumns.Employer:
+                    if (sortDescending)
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderByDescending(x => x.CertificateData.ContactOrganisation);
+                    }
+                    else
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderBy(x => x.CertificateData.ContactOrganisation);
+                    }
+
+                    break;
+                case GetCertificateHistoryRequest.SortColumns.ProviderName:
+                    if (sortDescending)
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderByDescending(x => x.CertificateData.ProviderName);
+                    }
+                    else
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderBy(x => x.CertificateData.ProviderName);
+                    }
+
+                    break;
+                case GetCertificateHistoryRequest.SortColumns.DateRequested:
+                    if (sortDescending)
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderByDescending(x => x.Certificate.CreatedAt);
+                    }
+                    else
+                    {
+                        certificatesQuery = certificatesQuery
+                            .OrderBy(x => x.Certificate.CreatedAt);
+                    }
+
+                    break;
+            }
+
+            return certificatesQuery;
+        }
+
         public async Task<int> GetCertificatesCount(string endPointAssessorOrganisationId, List<string> statuses)
```

Comment:
This looks like it was a count of what the get certificates used to do - is it still used?

Reply:
Removed